com.unity.ml-agents.extensions/Runtime/Sensors/PoseExtractor.cs

```diff
@@ -198,6 +203,17 @@ protected void Setup(int[] parentIndices)
         /// <returns></returns>
         protected internal abstract Vector3 GetLinearVelocityAt(int index);
 
+        /// <summary>
+        /// Return the underlying object at the given index. This is only
+        /// used for display in the inspector.
+        /// </summary>
+        /// <param name="index"></param>
+        /// <returns></returns>
+        protected internal virtual Object GetObjectAt(int index)
```

Comment:
abstract?

Reply:
It's meant to be overridden, but I think null is a reasonable default implementation.